java/com/scrat/everchanging/FireFlie.java

```diff
@@ -126,69 +234,96 @@ public class FireFlie extends TextureObject {
+            {{256, 256, 256, 242}, {0, 0, 0, 0}}, {{256, 256, 256, 235}, {0, 0, 0, 0}},
+            {{256, 256, 256, 228}, {0, 0, 0, 0}}, {{256, 256, 256, 222}, {0, 0, 0, 0}},
+            {{256, 256, 256, 215}, {0, 0, 0, 0}}, {{256, 256, 256, 208}, {0, 0, 0, 0}},
+            {{256, 256, 256, 202}, {0, 0, 0, 0}}, {{256, 256, 256, 196}, {0, 0, 0, 0}},
+            {{256, 256, 256, 189}, {0, 0, 0, 0}}, {{256, 256, 256, 183}, {0, 0, 0, 0}},
+            {{256, 256, 256, 177}, {0, 0, 0, 0}}, {{256, 256, 256, 171}, {0, 0, 0, 0}},
+            {{256, 256, 256, 165}, {0, 0, 0, 0}}, {{256, 256, 256, 159}, {0, 0, 0, 0}},
+            {{256, 256, 256, 153}, {0, 0, 0, 0}}, {{256, 256, 256, 148}, {0, 0, 0, 0}},
+            {{256, 256, 256, 142}, {0, 0, 0, 0}}, {{256, 256, 256, 136}, {0, 0, 0, 0}},
+            {{256, 256, 256, 131}, {0, 0, 0, 0}}, {{256, 256, 256, 126}, {0, 0, 0, 0}},
+            {{256, 256, 256, 121}, {0, 0, 0, 0}}, {{256, 256, 256, 116}, {0, 0, 0, 0}},
+            {{256, 256, 256, 111}, {0, 0, 0, 0}}, {{256, 256, 256, 106}, {0, 0, 0, 0}},
+            {{256, 256, 256, 101}, {0, 0, 0, 0}}, {{256, 256, 256,  96}, {0, 0, 0, 0}},
+            {{256, 256, 256,  91}, {0, 0, 0, 0}}, {{256, 256, 256,  86}, {0, 0, 0, 0}},
+            {{256, 256, 256,  82}, {0, 0, 0, 0}}, {{256, 256, 256,  78}, {0, 0, 0, 0}},
+            {{256, 256, 256,  73}, {0, 0, 0, 0}}, {{256, 256, 256,  69}, {0, 0, 0, 0}},
+            {{256, 256, 256,  65}, {0, 0, 0, 0}}, {{256, 256, 256,  61}, {0, 0, 0, 0}},
+            {{256, 256, 256,  57}, {0, 0, 0, 0}}, {{256, 256, 256,  53}, {0, 0, 0, 0}},
+            {{256, 256, 256,  49}, {0, 0, 0, 0}}, {{256, 256, 256,  46}, {0, 0, 0, 0}},
+            {{256, 256, 256,  42}, {0, 0, 0, 0}}, {{256, 256, 256,  38}, {0, 0, 0, 0}},
+            {{256, 256, 256,  35}, {0, 0, 0, 0}}, {{256, 256, 256,  32}, {0, 0, 0, 0}},
+            {{256, 256, 256,  28}, {0, 0, 0, 0}}, {{256, 256, 256,  25}, {0, 0, 0, 0}},
+            {{256, 256, 256,  22}, {0, 0, 0, 0}}, {{256, 256, 256,  19}, {0, 0, 0, 0}},
+            {{256, 256, 256,  16}, {0, 0, 0, 0}}, {{256, 256, 256,  13}, {0, 0, 0, 0}},
+            {{256, 256, 256,  10}, {0, 0, 0, 0}}, {{256, 256, 256,   8}, {0, 0, 0, 0}},
+            {{256, 256, 256,   5}, {0, 0, 0, 0}}, {{256, 256, 256,   2}, {0, 0, 0, 0}},
+            {{256, 256, 256,   0}, {0, 0, 0, 0}},
     };
-    int numClips = minObjects;
-    int frameCounter = 0;
-    int maxFrames = 6;
-    boolean init = false;
+    // @formatter:on
+
+    private static final int MAX_FRAMES = 6;
+
+    private int numClips = minObjects;
+    private int frameCounter = 0;
+    private boolean init = false;
 
     private final Calendar calendar;
 
-    static final String[][] textureList = {{"image_168", "image_170"}};
-    static final float[][] pivotList = {{7.5f, 7.5f}, {7.5f, 7.5f}};
+    private static final String[][] textureList = {{"image_168", "image_170"}};

```

Comment:
Seems I missed this class when reformatting code and changing visibility modifiers.
I added similar change in this PR.